lib/screens/VeggieDetail.dart

```diff
@@ -57,34 +57,92 @@ class _VeggieDetailState extends State<VeggieDetail> {
     });
   }
 
+  Widget _regularComponent(Widget servingInfo) {
+    return Column(
+      children: [
+        Text(
+          "SERVING INFO",
+          style: $styles.text.heading3.copyWith(color: $styles.colors.black),
+        ),
+        $styles.spacers.xs,
+        Container(
+          padding: EdgeInsets.all($styles.padding.m),
+          decoration: BoxDecoration(
+            color: $styles.colors.white,
+            borderRadius: BorderRadius.circular(20),
+            border: $styles.borderStyle.border,
+          ),
+          child: Column(
+            children: [
+              servingInfo,
+              $styles.spacers.m,
+              FutureBuilder(
+                future: _getCalorie(),
+                initialData: 2000,
+                builder: (context, snapshot) {
+                  return Text(
+                    "Percent daily values based on a diet of ${snapshot.data} calories",
+                    style: $styles.text.paragraph.copyWith(color: $styles.colors.grey),
+                  );
+                },
+              ),
+            ],
+          ),
+        ),
+        $styles.spacers.m,
+        Row(children: [
+          FutureBuilder(
+            future: _checkGarden(),
+            builder: (BuildContext context, AsyncSnapshot<bool> snapshot) {
+              if (snapshot.hasData) {
+                debugPrint("data ${snapshot.data!}");
+                return CustomSwitch(
+                  value: snapshot.data!,
+                  onChanged: (value) {
+                    setState(() {
+                      _setGarden(value);
+                    });
+                  },
+                );
+              }
+              return const Center(child: CircularProgressIndicator());
+            },
+          ),
+          $styles.spacers.xs,
+          Text(
+            "Save to garden",
+            style: $styles.text.label.copyWith(color: $styles.colors.grey),
+          )
+        ]),
+      ],
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
     var servingWords = widget.veggie.servingSize.split(" ");
 
-    var servingInfo =
-        Row(mainAxisAlignment: MainAxisAlignment.spaceBetween, children: [
-      Expanded(
-        flex: 1,
```

Comment:
flex: 1 is the default, no need to specify on all of these.
A lint would have caught this.

Reply:
Just so we have as a reference, the lint is `avoid_redundant_argument_values` but that's not included in flutter_lints atm. Kate's opened an issue here: https://github.com/dart-lang/lints/issues/170